Detections/DSMAzureBlobStorageLogs/AccessToSensitiveDataFromKnownTiMatchedIpAdresses.yaml

```diff
@@ -0,0 +1,86 @@
+id: e3c63c74-1140-406b-bfd6-9ddee2677f9e
+name: Access to sensitive data from known TI matched IP addresses
+description: |
+  This query detects access to data that has classification insights (link below) and/or sensitivity label insights (link below), from IP adresses that are marked via the TI.
```

Comment:
Description section starts and ends with a single quote. Please put it there.
Also, if we have links in description, please use Reference before it. See example below:
https://github.com/Azure/Azure-Sentinel/blob/d8ba6590bf4995b2fa1869027b47df93b60ebfd4/Detections/MultipleDataSources/BariumDomainIOC112020.yaml
Also remove localization i.e en-us part from the link as that might cause validation errors that the PR is currently getting.

Reply:
That's probably one of the two warnings I did not understand. Thanks

Reply:
done